src/stylus/components/_lists.styl

```diff
@@ -62,7 +63,8 @@ theme(v-list, "v-list")
     transition: $primary-transition
     user-select: none
 
-    &--link
+    &--link,
+    &--hover
```

Comment:
I'm not sure about this, it probably should just change the background but cursor shouldn't change as there is no any action

Reply:
Yeah I wasn't sure about this either. If there's any action `@click` would add the `cursor: pointer` anyway. Reverting.